src/__tests__/analytics-test.ts

```diff
@@ -139,6 +140,81 @@ test('log event in iOS', async () => {
     });
 });
 
+test('log ecommerce event in Android', async () => {
+    const androidFirebaseMock = givenAndroidWebview();
+
+    await logEcommerceEvent('select_item', {
+        item_list_id: 'L001',
+        item_list_name: 'Related products',
+        items: [
+            {
+                item_id: 'SKU_123',
+                item_name: 'jeggings',
+                item_category: 'pants',
+                item_variant: 'black',
+                item_brand: 'Google',
+                price: 9.99,
```

Comment:
this will be a string, but as it is typed as "any" we should be fine

Reply:
I just copied this example from google documentation, but I'd guess a string would work too